### PR TITLE
TE-219 OwnerSignUp widget

### PR DESCRIPTION
**Pull request by marcdavi-es**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-219)

### What **one** thing does this PR do?
Create component, tests and examples for OwnerSignUp

### CODE REVIEW

File: src/components/widgets/OwnerSignUp/component.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <TextInput label="First Name" name="firstName" />
+    <TextInput label="Last Name" name="lastName" />
+    <TextInput label="Email" name="email" />
+  </Form>
+);
+
+Component.displayName = 'OwnerSignUp';
+
+Component.defaultProps = {
+  onSubmit: Function.prototype,
```
Comment (matthizou) [inline src/components/widgets/OwnerSignUp/component.js]: Never seen that before, using the function prototype as an empty function ( () => {} ).  But yes, why not. Interesting

File: src/components/widgets/OwnerSignUp/component.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Form } from '../../collections/Form';
+import { TextInput } from '../../elements/TextInput';
+
+export const Component = ({ onSubmit }) => (
```
Comment (matthizou) [inline src/components/widgets/OwnerSignUp/component.js]: Missing description

File: src/components/widgets/OwnerSignUp/component.js (hunk 1 above)
Comment (marcdavi-es) [reply src/components/widgets/OwnerSignUp/component.js]: I think this is better for a noop. The empty function: 
1. creates a new reference which we don't need
2. gets picked up by test coverage

File: src/components/widgets/OwnerSignUp/component.js (hunk 2 above)
Comment (marcdavi-es) [reply src/components/widgets/OwnerSignUp/component.js]: Fixed

**Pull request merged**